src/test/java/com/puppycrawl/tools/checkstyle/xpath/RootNodeTest.java

```diff
@@ -110,6 +111,24 @@ public void testGetColumnNumber() {
             .isEqualTo(0);
     }
 
+    /**
+     * This test exists to cover pitest mutation.
+     * It is impossible to create RootNode that does not have column as 0.
+     * Test exists until https://github.com/checkstyle/checkstyle/issues/4997
```

Comment:
Inspections do not like raw links.
https://github.com/checkstyle/checkstyle/blob/c69d3aeb68da4215fc91bbf2e95b745b9f373994/src/test/java/com/puppycrawl/tools/checkstyle/XMLLoggerTest.java#L402

We should use non linked form of until.

Reply:
Or keep it as non javadoc
https://github.com/checkstyle/checkstyle/blob/c69d3aeb68da4215fc91bbf2e95b745b9f373994/src/test/java/com/puppycrawl/tools/checkstyle/api/FileContentsTest.java#L275-L277